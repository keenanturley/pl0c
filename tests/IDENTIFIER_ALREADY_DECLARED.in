var x, w, x;
begin
   x:= 4;
   w:= 7;
   if w > x then
	w:= w + 1;
	x:= w;
end
