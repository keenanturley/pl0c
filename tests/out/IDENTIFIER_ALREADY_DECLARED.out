Lexeme List:
29 2 x 17 2 w 17 2 x 18 21 2 x 20 3 4 18 2 w 20 3 7 18 23 2 w 13 2 x 24 2 w 20 2 w 4 3 1 18 2 x 20 2 w 18 22 

Lexeme List (Symbolic Representation):
varsym identsym x commasym identsym w commasym identsym x semicolonsym beginsym identsym x becomessym numbersym 4 semicolonsym identsym w becomessym numbersym 7 semicolonsym ifsym identsym w gtrsym identsym x thensym identsym w becomessym identsym w plussym numbersym 1 semicolonsym identsym x becomessym identsym w semicolonsym endsym 

Error: Attempted to redeclare existing identifier.
